releasenotes/notes/v05-encoding-eac70f23850a445e.yaml

```
@@ -0,0 +1,4 @@
+---
+upgrade:
+  - |
+    tracing: Upgrade the default trace API version to ``v0.5``.
```

```suggestion
    tracing: upgrade the default trace API version to ``v0.5``.
```

can we extend this note to include how v0.5 is more performant especially with larger traces?